src/main/java/com/simanavets/tenderflexbackend/entity/Contract.java

```diff
@@ -0,0 +1,27 @@
+package com.simanavets.tenderflexbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Contract {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
```

Comment:
it's not many to one because each contract can only have relation with one tender and one offer.